src/protagonist/Portal/Pages/Queue/Upload.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@page
+@model Portal.Pages.Queue.IndexModel;
+
+@{
+    ViewData["Title"] = "Upload";
+}
+
+@section header
+{
+    <link href="https://cdn.jsdelivr.net/npm/dropzone@5.9.2/dist/min/dropzone.min.css" rel="stylesheet" />
+    <link href="~/assets/dist/css/jquery-ui.min.css" rel="stylesheet" />
+    <link href="~/assets/dist/css/jquery-ui.structure.min.css" rel="stylesheet" />
+}
+
+<div>On this page you can upload a CSV of information about images for the DLCS to ingest.</div>
+<a href="https://legacyportal.dlcs.digirati.io/Support/sample.csv">Click here to download a sample .csv file.</a>
+<div class="alert alert-danger my-2" id="upload-error-box" role="alert" hidden>
+    <i class="me-2" data-feather="alert-circle"></i><strong>Error!</strong>
+    <ul class="my-1" id="errors">
+    </ul>
+</div>
+<div class="my-2">
+    <form action="@Url.Action("Upload", "CsvUpload", new { customer = 27, space = 1 })" class="dropzone" id="dropzoneForm"></form>
```

Comment:
Hardcoded route values

Reply:
I forgot about these. They're not used any more either, so I'll just remove them